### ISSUE TITLE
Feature request: Hide "Inbox" in Multiple Inbox setup

### ISSUE THREAD

**Issue by drwb** [leggett/simplify#647]
Would it be possible to have an option to hide the default Inbox when in Multiple Inbox mode?  The choices in the Gmail settings are Right of the inbox, Above the inbox, Below the inbox but having "Do not show" would be great - that way I could create another inbox of "Anything Else" with custom filters which should, all being well, be empty.

Graphically, I'd like to hide the red section below:

<img width="1501" alt="Screenshot 2022-01-06 at 19 01 25" src="https://user-images.githubusercontent.com/16479230/148437882-df7711ce-20cc-4da2-8330-74ac4a6a87c1.png">

Thanks!

**Comment by leggett** [comment]
I have an option to hide Multiple Inbox sections when empty and I could extend it to also hide the Inbox section itself when empty. But it isn't empty in your screenshot.

Having an option to hide it always is risky as it depends on you setting up inbox searches that cover everything. And searches don't always work as you might expect because they are message-based while the interface is thread-based.

Maybe an option to minimize it instead of hiding it all together?

**Comment by drwb** [comment]
Indeed, it isn’t empty, but everything in it is sorted in the above Inboxes, but I take your point on the dangers.

The ability to minimise it would work. That way I can clear out the above sections and then do a double check in the Inbox.

Thanks!

**Comment by leggett** [comment]
Sounds good. I'll put it on my list of things to look into.
